docs/src/pages/components/data-grid/editing/ValidateCellApiRefGrid.js

```diff
@@ -27,8 +27,7 @@ const useStyles = makeStyles(
 );
 
 function validateEmail(email) {
-  const re =
-    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  const re = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
```

Comment:
This line shouldn't appear as a change. I faced the same situation in another PR but it was because I was using an old version of prettier. It was updated in #1843. Running `yarn` and `yarn prettier` is enough.